### PR TITLE
M3-625 Display additional networking information.

### PR DESCRIPTION
**Pull request by rjstires**
## Purpose
Display additional networking related data to the user.
- Added link-local IP.
- Added DNS Resolvers, shows one IPv4 and one IPv6 and the rest in a popover.
- Added SSH and Lish via SSH links (copyable).
- Added default gateways for public IPs and IPv6.

### CODE REVIEW

File: src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+const styled = withStyles(styles, { withTheme: true });
+
+const connected = connect((state: Linode.AppState) => ({
+  username: path(['resources', 'profile', 'data', 'username'], state),
+}));
+
+export default styled(connected(LinodeNetworkingSummaryPanel)) as React.ComponentType<Props>;
+
+const renderLinkLocak = (linkLocal: string) => () =>
```
Comment (martinmckenna) [inline src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx]: is this supposed to be `renderLinkLocal`?

**Comment by Jskobos** [comment]
SSH text alignment is a little off in Chrome:

screen shot 2018-06-22 at 4 20 09 pm

File: src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    <Typography variant="caption" style={{ display: 'inline-block' }}>ssh root@{address}</Typography>
+    <CopyTooltip text={`ssh root@{address}`} />
+  </React.Fragment>
+
+const renderLishLink = (username: string, region: string, linodeLabel: string) => () =>
+  <React.Fragment>
+    <Typography variant="caption" style={{ display: 'inline-block' }}>
+      ssh -t {username}@lish-{region}.linode.com {linodeLabel}
+    </Typography>
+    <CopyTooltip text={`ssh -t {username}@lish-{region}.linode.com {linodeLabel}`} />
```
Comment (Jskobos) [inline src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx]: I think this needs some `$`, as is the copied link doesn't have the resolved variables in it.

File: src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    <ShowMore items={tail(ipv6DNSResolvers)} render={renderAddresses} />
+  </React.Fragment>);
+
+const renderAddresses = (addresses: string[]) => addresses.map(a =>
+  <Typography variant="caption" key={a} style={{ marginBottom: '4px', marginTop: '4px' }}>{a}</Typography>)
+
+const renderSSHLink = (address?: string) => () =>
+  <React.Fragment>
+    <Typography variant="caption" style={{ display: 'inline-block' }}>ssh root@{address}</Typography>
+    <CopyTooltip text={`ssh root@{address}`} />
```
Comment (Jskobos) [inline src/features/linodes/LinodesDetail/LinodeNetworking/LinodeNetworkingSummaryPanel.tsx]: Needs `$` (same as below)

**Pull request merged**